documentation/requirements.txt

```diff
@@ -1,10 +1,11 @@
 # Python dependencies to build the documentation
 
 # != 3.5.0: https://github.com/sphinx-doc/sphinx/issues/8880
-Sphinx ~= 3.1, != 3.5.0
+Sphinx ~= 4.0, != 3.5.0
 breathe
 sphinx-rtd-theme
 cairosvg
+sphinxcontrib-svg2pdfconverter
```

Comment:
`sphinxcontrib-svg2pdfconverter[CairoSVG]`

And remove the cairosvg dependency above.

Reply:
```suggestion
sphinxcontrib-svg2pdfconverter[CairoSVG]
```